### ISSUE TITLE
Intel 600p random crashing (Monterey) / NVMEfix

### ISSUE THREAD

**Issue by Thiadeth** [acidanthera/bugtracker#1961]
Can you fix this? I crashed so many times cause of it, otherwise a very stable hack build, I even got iMessage and FaceTime working.

- Only the latest released versions of OpenCore, Lilu, and other KEXTs are supported.
latest version, open core debug, kexts are release versions
- Include your your `EFI` directory with OpenCore.
[EFI.ZIP](https://www.icloud.com/iclouddrive/01ba76dgLw3Jq6H6aglAFkI-A#EFI)
- Include OpenCore DEBUG log (refer to `Target` in Configuration.pdf).
Included in EFI zip

```
panic(cpu 0 caller 0xffffff800773b836): nvme: "3rd party NVMe controller. Loss of MMIO space. Write. fBuiltIn=1 MODEL=INTEL SSDPEKKW256G7 FW= PSF109C CSTS=0xffffffff US[1]=0x0 US[0]=0x16b VID=0x8086 DID=0xf1a5 CRITICAL_WARNING=0x0.\n" @IONVMeController.cpp:6053
Panicked task 0xffffff8690775670: 177 threads: pid 0: kernel_task
Backtrace (CPU 0), panicked thread: 0xffffff94f78e4000, Frame : Return Address
0xffffffe0637bbaa0 : 0xffffff8004e85ffd mach_kernel : _handle_debugger_trap + 0x41d
0xffffffe0637bbaf0 : 0xffffff8004fe6035 mach_kernel : _kdp_i386_trap + 0x145
0xffffffe0637bbb30 : 0xffffff8004fd5803 mach_kernel : _kernel_trap + 0x533
0xffffffe0637bbb80 : 0xffffff8004e25a60 mach_kernel : _return_from_trap + 0xe0
0xffffffe0637bbba0 : 0xffffff8004e863cd mach_kernel : _DebuggerTrapWithState + 0xad
0xffffffe0637bbcc0 : 0xffffff8004e85b86 mach_kernel : _panic_trap_to_debugger + 0x2b6
0xffffffe0637bbd20 : 0xffffff8005716409 mach_kernel : _panic + 0x54
0xffffffe0637bbd90 : 0xffffff800773b836 com.apple.iokit.IONVMeFamily : __ZN16IONVMeController14CommandTimeoutEP16AppleNVMeRequest.cold.1
0xffffffe0637bbda0 : 0xffffff800771eac7 com.apple.iokit.IONVMeFamily : __ZN16IONVMeController13FatalHandlingEv + 0x141
0xffffffe0637bbdd0 : 0xffffff800565dd25 mach_kernel : __ZN18IOTimerEventSource15timeoutSignaledEPvS0_ + 0xa5
0xffffffe0637bbe40 : 0xffffff800565dc28 mach_kernel : __ZN18IOTimerEventSource17timeoutAndReleaseEPvS0_ + 0xc8
0xffffffe0637bbe70 : 0xffffff8004ed93a5 mach_kernel : _thread_call_delayed_timer + 0x4f5
0xffffffe0637bbee0 : 0xffffff8004eda412 mach_kernel : _thread_call_delayed_timer + 0x1562
0xffffffe0637bbfa0 : 0xffffff8004e2518e mach_kernel : _call_continuation + 0x2e
      Kernel Extensions in backtrace:
         com.apple.iokit.IONVMeFamily(2.1)[0DE31F96-AEED-31F5-84D4-4147BDC77799]@0xffffff8007716000->0xffffff8007742fff
            dependency: com.apple.driver.AppleMobileFileIntegrity(1.0.5)[B0319B71-F6E6-39E1-AD7C-33CE7A82BDEC]@0xffffff800656c000->0xffffff800658dfff
            dependency: com.apple.iokit.IOPCIFamily(2.9)[897B72E0-B98F-30BA-8CB2-4E5E469CE4B6]@0xffffff80079e7000->0xffffff8007a11fff
            dependency: com.apple.iokit.IOReportFamily(47)[FCE9B54C-4E85-371D-BB93-1B9AA324711A]@0xffffff8007a23000->0xffffff8007a25fff
            dependency: com.apple.iokit.IOStorageFamily(2.1)[160B70B8-9236-38ED-8AC9-A215D071A76B]@0xffffff8007b28000->0xffffff8007b3efff

Process name corresponding to current thread (0xffffff94f78e4000): kernel_task
Boot args: -v keepsyms=1 debug=0x100 alcid=1 chunklist-security-epoch=0 -chunklist-no-rev2-dev

Mac OS version:
21D62

Kernel version:
Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_X86_64
Kernel UUID: 93729D02-FE6F-355B-BA76-BA930AA7103F
KernelCache slide: 0x0000000004c00000
KernelCache base:  0xffffff8004e00000
Kernel slide:      0x0000000004c10000
Kernel text base:  0xffffff8004e10000
__HIB  text base: 0xffffff8004d00000
System model name: MacBookPro14,2 (Mac-CAD6701F7CEA0921)
System shutdown begun: NO
Panic diags file available: YES (0x0)
Hibernation exit count: 0

System uptime in nanoseconds: 75189599643
Last Sleep:           absolute           base_tsc          base_nano
  Uptime  : 0x0000001181a64193
  Sleep   : 0x0000000000000000 0x0000000000000000 0x0000000000000000
  Wake    : 0x0000000000000000 0x000000059b6cfce3 0x0000000000000000
Zone info:
Foreign   : 0xffffff801d8a0000 - 0xffffff801d8ad000
Native    : 0xffffff8029e89000 - 0xffffffa029e89000
Readonly  : 0xffffff84f6b55000 - 0xffffff86904e9000
Metadata  : 0xfffffff6fda88000 - 0xfffffff71db4e000
Bitmaps   : 0xfffffff71db4e000 - 0xfffffff720b4e000
```

Edit: it happened again, I think more often when I mount refi to update config.plist

```
panic(cpu 0 caller 0xffffff801593b836): nvme: "3rd party NVMe controller. Loss of MMIO space. Write. fBuiltIn=1 MODEL=INTEL SSDPEKKW256G7 FW= PSF109C CSTS=0xffffffff US[1]=0x0 US[0]=0x16b VID=0x8086 DID=0xf1a5 CRITICAL_WARNING=0x0.\n" @IONVMeController.cpp:6053
Panicked task 0xffffff9038317670: 182 threads: pid 0: kernel_task
Backtrace (CPU 0), panicked thread: 0xffffff8b64769000, Frame : Return Address
0xffffffe069733aa0 : 0xffffff8013085ffd mach_kernel : _handle_debugger_trap + 0x41d
0xffffffe069733af0 : 0xffffff80131e6035 mach_kernel : _kdp_i386_trap + 0x145
0xffffffe069733b30 : 0xffffff80131d5803 mach_kernel : _kernel_trap + 0x533
0xffffffe069733b80 : 0xffffff8013025a60 mach_kernel : _return_from_trap + 0xe0
0xffffffe069733ba0 : 0xffffff80130863cd mach_kernel : _DebuggerTrapWithState + 0xad
0xffffffe069733cc0 : 0xffffff8013085b86 mach_kernel : _panic_trap_to_debugger + 0x2b6
0xffffffe069733d20 : 0xffffff8013916409 mach_kernel : _panic + 0x54
0xffffffe069733d90 : 0xffffff801593b836 com.apple.iokit.IONVMeFamily : __ZN16IONVMeController14CommandTimeoutEP16AppleNVMeRequest.cold.1
0xffffffe069733da0 : 0xffffff801591eac7 com.apple.iokit.IONVMeFamily : __ZN16IONVMeController13FatalHandlingEv + 0x141
0xffffffe069733dd0 : 0xffffff801385dd25 mach_kernel : __ZN18IOTimerEventSource15timeoutSignaledEPvS0_ + 0xa5
0xffffffe069733e40 : 0xffffff801385dc28 mach_kernel : __ZN18IOTimerEventSource17timeoutAndReleaseEPvS0_ + 0xc8
0xffffffe069733e70 : 0xffffff80130d93a5 mach_kernel : _thread_call_delayed_timer + 0x4f5
0xffffffe069733ee0 : 0xffffff80130da412 mach_kernel : _thread_call_delayed_timer + 0x1562
0xffffffe069733fa0 : 0xffffff801302518e mach_kernel : _call_continuation + 0x2e
      Kernel Extensions in backtrace:
         com.apple.iokit.IONVMeFamily(2.1)[0DE31F96-AEED-31F5-84D4-4147BDC77799]@0xffffff8015916000->0xffffff8015942fff
            dependency: com.apple.driver.AppleMobileFileIntegrity(1.0.5)[B0319B71-F6E6-39E1-AD7C-33CE7A82BDEC]@0xffffff801476c000->0xffffff801478dfff
            dependency: com.apple.iokit.IOPCIFamily(2.9)[897B72E0-B98F-30BA-8CB2-4E5E469CE4B6]@0xffffff8015be7000->0xffffff8015c11fff
            dependency: com.apple.iokit.IOReportFamily(47)[FCE9B54C-4E85-371D-BB93-1B9AA324711A]@0xffffff8015c23000->0xffffff8015c25fff
            dependency: com.apple.iokit.IOStorageFamily(2.1)[160B70B8-9236-38ED-8AC9-A215D071A76B]@0xffffff8015d28000->0xffffff8015d3efff

Process name corresponding to current thread (0xffffff8b64769000): kernel_task
Boot args: -v keepsyms=1 debug=0x100 alcid=1 chunklist-security-epoch=0 -chunklist-no-rev2-dev

Mac OS version:
21D62

Kernel version:
Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_X86_64
Kernel UUID: 93729D02-FE6F-355B-BA76-BA930AA7103F
KernelCache slide: 0x0000000012e00000
KernelCache base:  0xffffff8013000000
Kernel slide:      0x0000000012e10000
Kernel text base:  0xffffff8013010000
__HIB  text base: 0xffffff8012f00000
System model name: MacBookPro14,2 (Mac-CAD6701F7CEA0921)
System shutdown begun: NO
Panic diags file available: YES (0x0)
Hibernation exit count: 0

System uptime in nanoseconds: 705108505823
Last Sleep:           absolute           base_tsc          base_nano
  Uptime  : 0x000000a42bbdf8d5
  Sleep   : 0x0000000000000000 0x0000000000000000 0x0000000000000000
  Wake    : 0x0000000000000000 0x00000005d1b2c292 0x0000000000000000
Zone info:
Foreign   : 0xffffff802baa0000 - 0xffffff802baad000
Native    : 0xffffff8038190000 - 0xffffffa038190000
Readonly  : 0xffffff8504e5c000 - 0xffffff869e7f0000
Metadata  : 0xfffffffcb12be000 - 0xfffffffcd1385000
Bitmaps   : 0xfffffffcd1385000 - 0xfffffffcd4385000
```

**Comment by antoniomcr96** [comment]
That disk isn't a good choice
- https://github.com/acidanthera/bugtracker/issues/1286

**Comment by Thiadeth** [comment]
> That disk isn't a good choice
> 
> * [Any fix for Intel 600p NVMe Drive? #1286](https://github.com/acidanthera/bugtracker/issues/1286)

Yeah but i dont want to buy a new ssd, and i dont have another pc with nvme slots to clone it.... but i am more than willing to troubleshoot the heck out of it and send all the log files needed

**Comment by icyphox** [comment]
I've got the same issue on my Intel 600p SSD—I'm happy to provide any logs/info to try and fix this.
